recipes/azure-common/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.1.3" %}
+
+package:
+  name: azure-common
+  version: {{ version }}
+
+source:
+  fn: azure-common-{{ version }}.zip
+  url: https://pypi.python.org/packages/48/89/017df9bed55925de004076937ca168d3a03aa0250fbda7826ec3c0ac714f/azure-common-{{ version }}.zip
```

Comment:
Could we please change this to `https://pypi.io/packages/source/a/azure-common/azure-common-{{ version }}.zip`?